internal/core/model/adapters.rs

```diff
@@ -343,9 +351,17 @@ where
 
     /// Manually reapply the filter. You need to run this e.g. if the filtering function compares
     /// against mutable state and it has changed.
+    #[deprecated(note = "Use reset() instead")]
     pub fn apply_filter(&self) {
+        self.reset()
+    }
+
+    /// Manually reapply the filter. You need to run this e.g. if the filtering function compares
+    /// against mutable state and it has changed.
```

Comment:
I think it should be "uses mutable state" instead of "compare against"

Reply:
Sounds good. It's done